### PR TITLE
Cndit 1568 : Updating the Liquibase Build to ECR github actions 

### PR DESCRIPTION
**Pull request by alog-enquizit**
Updating the Liquibase Build to ECR github actions

### CODE REVIEW

File: .github/workflows/Build-and-deploy-reporting-services.yaml [hunk 1]
```diff
@@ -6,107 +6,107 @@ on:
...
-      dockerfile_relative_path: -f ./ldfdata-service/Dockerfile .
-      environment_classifier: SANDBOX
-      java_version: "17"
-    secrets:
-      CDC_NBS_SANDBOX_SHARED_SERVICES_ACCOUNTID: ${{secrets.CDC_NBS_SANDBOX_SHARED_SERVICES_ACCOUNTID}}
-      ECR_REPO_BASE_NAME: ${{secrets.ECR_REPO_BASE_NAME}}
-      GIT_USER_EMAIL: ${{secrets.GIT_USER_EMAIL}}
-      GIT_USER_NAME: ${{secrets.GIT_USER_NAME}}
-      HELM_TOKEN: ${{secrets.HELM_TOKEN}}
+#  call-build-person-reporting-microservice-container-workflow:
```
Comment (alog-enquizit) [inline .github/workflows/Build-and-deploy-reporting-services.yaml:16]: Removing the github actions to build docker image from this repo for the RTR services as we have moved them to the new repo.

**Review by upasanapattnaik-eq — COMMENTED** [review]
This PR is just to remove the github actions, correct? No additions from here on

**Review by upasanapattnaik-eq — APPROVED** [review]
LGTM!

**Comment by alog-enquizit** [comment]
> This PR is just to remove the github actions, correct? No additions from here on

@upasanapattnaik-eq Also to make it deployable from the kubernetes EKS pods

**Pull request merged**